Sources/ReaderDocument.m

```diff
@@ -257,21 +260,6 @@ - (NSURL *)fileURL
 	return _fileURL;
 }
 
-- (BOOL)canEmail
-{
-	return YES;
-}
-
-- (BOOL)canExport
-{
-	return YES;
-}
-
-- (BOOL)canPrint
-{
-	return YES;
-}
-
```

Comment:
Maybe they should default to be true (make the least breaking change possible) :)